etc/metrics_metadata.json

```diff
@@ -839,6 +923,18 @@
         ],
         "stability": "committed"
     },
+    "xdcr_target_unknown_status": {
+        "type": "counter",
+        "help": "The total number of writes to target KV that returned with a status code that XDCR cannot comprehend",
```

Comment:
We use "data service" instead of KV in description. See related stats messages

Reply:
I will change all "KV" to "data service". But this is is already there before my changes.